_posts/2023-11-21-trino-summit-2023-nears-lineup.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: post
+title: "Trino Summit 2023 nears with an awesome lineup"
+author: "Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2023/lineup-blog-banner.png
+---
+
+As winter nears, the days may be getting shorter, but so is the wait until
+Trino Summit 2023! It'll be here before you know it on December 13th and 14th,
+and we've got a packed speaker lineup full of exciting talks that we're ready to
+share with the Trino community today. Read on for a preview of all the talks
+we're looking forward to, and if you're interested in attending, make sure to...
```

Comment:
```suggestion
, and if you're interested in attending, make sure to ...
```

Reply:
The space before the ellipsis isn't necessarily correct. Every style guide differs on its recommendations, but I've found the lack of space before trailing off to be the most commonly-accepted approach in informal writing. See, for example, the [Buzzfeed Style Guide](https://www.buzzfeed.com/buzzfeednews/buzzfeed-style-guide).

Reply:
no worries